hudi-client/hudi-spark-client/pom.xml

```diff
@@ -174,6 +194,12 @@
       <artifactId>awaitility</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.thoughtworks.paranamer</groupId>
+      <artifactId>paranamer</artifactId>
+      <version>2.8</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
How is this introduced?  Does it have a compatible OSS license?

Reply:
My assumption is you are referring to this dependency https://github.com/paul-hammant/paranamer?

    <dependency>
      <groupId>com.thoughtworks.paranamer</groupId>
      <artifactId>paranamer</artifactId>
      <version>2.8</version>
      <scope>test</scope>
    </dependency>


this change was made by @xushiyan https://issues.apache.org/jira/browse/HUDI-3088 #4752 but i think we need this newer 2.8 for spark 3 if its the default profile. In master we have this https://github.com/apache/hudi/search?q=paranamer in the licenses we seem to be referring to paranamer 2.7

here are some jiras ive seen discussing the issue https://github.com/apache/beam/pull/17424  https://issues.apache.org/jira/browse/BEAM-14345